dev_tools/qualtran_dev_tools/jupyter_autogen_v2.py

```diff
@@ -52,17 +52,17 @@ class NotebookSpecV2:
     module: ModuleType
     bloq_specs: List[BloqDocSpec]
     directory: str = field()
-    _path_stem: Optional[str] = None
+    path_stem_name: Optional[str] = None
```

Comment:
why this change? attrs will strip the leading underscore to accept this as a init arg

Reply:
That's correct, this is not needed. I've reverted the change. Thanks for flagging.